web/installmanager/index.js

```diff
@@ -6,13 +10,36 @@
         const row = document.createElement('tr');
         const nameCell = document.createElement('td');
         const goCell = document.createElement('td');
+        const buttonsDiv = document.createElement('div');
+
+        buttonsDiv.style.display = 'flex';
+        buttonsDiv.style.gap = '10px';
+        buttonsDiv.style.alignItems = 'center';
+        buttonsDiv.style.justifyContent = 'center';
 
         goCell.style.textAlign = 'center';
 
         console.log(JSON.stringify(install));
 
         const nameContainer = document.createElement('div');
 
+        let editablespan = document.createElement('input');
+        editablespan.type = 'text';
+        editablespan.style.display = 'block';
+        editablespan.style.margin = '0';
+        editablespan.style.height = '22px';
+        editablespan.style.fontSize = '13px';
+        editablespan.value = sanitizeHTML(install.name || `Install #${install.index + 1}`);
+        editablespan.style.cursor = 'text';
+        editablespan.onblur = () => {
+            if (countingSake(editablespan.value.trim()) == "") {
+                window.alert("Installation name cannot be empty.");
+                editablespan.value = `Install #${install.index + 1}`;
+            }
+            install.name = editablespan.value.trim();
+            window.electronAPI.invoke('setInstallationCName', [""+install.index, install.name]);
```

Comment:
```suggestion
            window.electronAPI.invoke('setInstallationCName', [install.index.toString(), install.name]);
```
please. this hurts my eyes.